tracardi/process_engine/action/v1/strings/string_join/plugin.py

```diff
@@ -0,0 +1,77 @@
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Form, FormGroup, FormField, FormComponent, \
+    Documentation, PortDoc
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.result import Result
+from .model.configuration import Configuration
+
+
+def validate(config: dict) -> Configuration:
+    return Configuration(**config)
+
+
+class JoinAction(ActionRunner):
+    config: Configuration
+
+    async def set_up(self, init):
+        self.config = validate(init)
+
+    async def run(self, payload: dict, in_edge=None) -> Result:
+        dot = self._get_dot_accessor(payload)
+        string = dot[self.config.string]
+        result = self.config.delimiter.join(string)
```

Comment:
Please check if the string is an array. If it is not raise ValueError.  And I would name id data_to_join or something like this to be more clear.

Reply:
It's added at c1d01e9

Reply:
great